6_Lesson/Lesson6-1/Infrastructure/FabAccount.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace _6_Lesson;
+
+public static class FabAccount
+{
+    public static IEnumerable<string> EnumLines(this FileInfo file)
+    {
+        using var reader = file.OpenText();
+        while (!reader.EndOfStream)
+        {
+            var line = reader.ReadLine();
+            yield return line!;
+        }
```

Comment:
Можно красиве
```C#
        while (!reader.EndOfStream && reader.ReadLine() is { } line)
            yield return line;
```

Reply:
Можно даже упростить
```suggestion
        while (reader.ReadLine() is { } line)
            yield return line;
```